packages/app/src/TeamPage.tsx

```diff
@@ -38,6 +37,29 @@ export default function TeamPage() {
   const hasAllowedAuthMethods =
     team?.allowedAuthMethods != null && team?.allowedAuthMethods.length > 0;
 
+  const sentryDsn = useMemo(() => {
+    if (!team) {
+      return '';
+    }
+
+    try {
+      const ingestorUrl = new URL(process.env.NEXT_PUBLIC_INGESTOR_API_URL);
```

Comment:
This won't work with the prod build since process.env only works at dev env. Need to checkout how we inject custom env var at files including `app/src/config.ts`, `app/Dockerfile`, `otel-collector/config.yaml` and `Makefile`

Reply:
To verify the prod build is working, can run `make build-local` and `docker compose up -d`